### PR TITLE
Resource system

### PR DESCRIPTION
**Pull request by johnavalos2**

### CODE REVIEW

**Review by jalexcrawford — APPROVED** [review]
Looks good! 

I made a couple of comments, but I think this is good as is.

File: Assets/Scripts/Contract/Contract.cs [hunk 1]
```diff
@@ -20,11 +20,6 @@ public Contract(string contractName, float totalEffort, List<string> assignedWor
     {
         this.contractName = contractName;
         this.totalEffort = totalEffort;
-        // TODO: This needs to be addressed. Workers will not be assigned to contracts when they are created so we do not need this.
```
Comment (andrewcgraves) [inline Assets/Scripts/Contract/Contract.cs:23]: If this change was made, I would also suggest removing the entry for assigned workers in the constructor method as to not confuse the person using it.

File: Assets/Scripts/GameManager/Resources.cs [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+    public float GetTime()
+    {
+        return this.time;
+    }
+
+    public void SetEmployeeMorale(string employeeID, float newMorale)
+    {
+        this.morale[employeeID] = newMorale;
+    }
```
[the remark this answers is not in the document]
Comment (johnavalos2) [reply Assets/Scripts/GameManager/Resources.cs:69]: Per our conversation during the discord meeting we'll keep the morale under the resources controller for ease of use on the frontend, but may want to address this in the future to be a part of the employee controller.

File: Assets/Scripts/GameManager/Resources.cs [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+    public void SetWorkingHours(float newHours)
+    {
+        this.workingHours = newHours;
+    }
+
+    public float GetWorkingHours()
+    {
+        return this.workingHours;
+    }
```
[the remark this answers is not in the document]
Comment (johnavalos2) [reply Assets/Scripts/GameManager/Resources.cs:44]: Same train of thought with this method as mentioned in the above morale implementation.

**Review by andrewcgraves — APPROVED** [review]

**Pull request merged**